azure/packages/test/scenario-runner/testConfig_eastus.yml

```diff
@@ -0,0 +1,40 @@
+title: 10 new empty docs 20 loads eeach
+description: In this scenario we create 10 docs with no data, then just try to load them 20 times each
+env:
+    schema:
+        initialObjects:
+            key1: SharedMap
+            key2: SharedMap
+    connectionConfig:
+        type: remote
+        endpoint: https://alfred.eastus.fluidrelay.azure.com
```

Comment:
Given that we have to setup keys/secrets through env anyways, could we just pass endpoints through env also? Then you likely do not need separate yaml for each region.

Reply:
sounds like a good idea, will do 👍